### ISSUE TITLE
   A11y_Model Builder_azure_Input_Info and relationship: Screen reader is announcing read only for interactive control.

### ISSUE THREAD

**Issue by SachinChavhan317** [dotnet/machinelearning-modelbuilder#2655]
### GitHub Tags:
#WCAG1.3.1; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #Info and Relationships; #Screenreader; #Narrator; #FTP; #Win32; #Benchmark;

### Actual Result:
Screen reader is announcing read only for interactive control.

Note: Same issue observed with NVDA.

### Expected Result:
Screen reader should not announce read only for interactive control.

### Environment:
Application Name: Model Builder
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise Build 22624.1755
Screen reader: Narrator

### Repro Steps:

1. Launch the Visual Studio with at least one of the .NET workload.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu(Right-click) of the project in the solution explorer.
5. Tab till Add and then tab till Machine Learning Model and press enter.
6. Model Builder dialog will open. Tab till Scenario tab.
7. Select Scenario screen will appear. Tab till "Value Prection" tile and press enter.
8. Tab till Local CPU in environment and press enter to select it.
9. Turn on Screen reader.
10. Now tab till "
11. Now verify that screen reader is announcing read only for interactive control or not.

### User Impact:
Screen reader user will get confused if screen reader is announcing read only for an interactive controls.
### Attachment:
Narrator attachment
![Presentation1](https://github.com/dotnet/machinelearning-modelbuilder/assets/86241494/ac89316d-c56a-4b62-a8d8-30dcf1a5f282)

NVDA attachment
![Presentation2](https://github.com/dotnet/machinelearning-modelbuilder/assets/86241494/d91707c9-f925-4f76-82d5-daf78c2981c0)

**Comment by zewditu** [comment]
b/c this is set is read-only

**Comment by zewditu** [comment]
Use [this](https://microsoft.sharepoint.com/sites/PrivateBuildforAccessibility/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FPrivateBuildforAccessibility%2FShared%20Documents%2FMicrosoft%2EML%2EModelBuilder%20%2823%29%2Evsix&parent=%2Fsites%2FPrivateBuildforAccessibility%2FShared%20Documents) build to validate this issue
